mix.exs

```diff
@@ -1,24 +1,32 @@
 defmodule GGity.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/srowley/ggity"
+  @version "0.3.0"
+
   def project do
     [
       app: :ggity,
-      version: "0.3.0",
+      version: @version,
       aliases: aliases(),
       elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
       package: package(),
       deps: deps(),
       dialyzer: [plt_add_apps: [:mix]],
       name: "GGity",
-      description: "GGity brings the familiar interface of R's ggplot2 library to SVG charting in Elixir.",
-      source_url: "https://github.com/srowley/ggity",
```

Comment:
Per the `ExDoc` [docs](https://hexdocs.pm/ex_doc/readme.html#using-exdoc-with-mix), `:source_url` is a top-level option, not a key passed to the `:docs` option.

Reply:
> Per the `ExDoc` [docs](https://hexdocs.pm/ex_doc/readme.html#using-exdoc-with-mix), `:source_url` is a top-level option, not a key passed to the `:docs` option.

`:source_url` is [a key within the `:docs` option](https://github.com/elixir-lang/ex_doc/blob/master/lib/ex_doc/config.ex#L41) but due to [lack of awareness and workaround](https://www.kianmeng.org/2020/11/using-sourceurl-and-homepageurl-with.html) to it, it can be either at top-level or within the `:docs` itself. Either way, it works but the top-level option will override the nested option if exists.

I've moved to the top-level as it's a matter of preference. :smiley: